index.js

```diff
@@ -22,62 +23,62 @@ function getRandomNumber(min, max) {
   return Math.floor(Math.random() * (max - min)) + min;
 }
 
+function hideAllMessages() {
+  for (let elementIndex = 0; elementIndex < messages.length; elementIndex++) {
+    messages[elementIndex].style.display = 'none';
+  }
+}
+
 function checkGuess() {
   // Get value from guess input element
   const guess = parseInt(guessInput.value, 10);
   attempts = attempts + 1;
-
+  
   hideAllMessages();
 
-  if (guess === targetNumber) {
-    numberOfGuessesMessage.style.display = '';
-    numberOfGuessesMessage.innerHTML = `You made ${attempts} guesses`;
```

Comment:
If you get the answer right first then your page reads: "You made 1 guesses!". This isn't the kind of bug you would see in the console, but it's a bug in your user's experience. While it may be out of the scope of this exact assignment, I want to show you anyways:

You can run a ternary expression here to provide the proper grammar based on the number of attempts that have been made. For example:

`numberOfGuessesMessage.innerHTML = `You made ${attempts} guess${attempts.length === 1 ? '' : 'es'}`;

This would display You made 1 guess! If the user got it correct the first time, and "guesses" if attempts were less more more than 1.